optimum/onnxruntime/utils.py

```diff
@@ -271,3 +271,23 @@ class ORTQuantizableOperator(Enum):
     Resize = "Resize"
     AveragePool = "AveragePool"
     Concat = "Concat"
+
+
+def _get_external_data_paths(src_file_names, dst_file_names):
```

Comment:
I agree that we should change the naming to make it clear about the types.
Also add type annotation:
```suggestion
def _get_external_data_paths(src_paths: List[Path], dst_file_names: List[str]) -> Tuple[List[Path], List[str]]:
```